media-libs/lcms/lcms-2.9.recipe

```diff
@@ -23,14 +23,17 @@ if [ "$targetArchitecture" = x86_gcc2 ]; then
 	commandBinDir=$prefix/bin
 fi
 
+libVersion="2.0.8"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
-	lcms$secondaryArchSuffix = $portVersion compat >= 2
-	cmd:jpgicc$commandSuffix
-	cmd:linkicc$commandSuffix
-	cmd:psicc$commandSuffix
-	cmd:tificc$commandSuffix
-	cmd:transicc$commandSuffix
-	lib:liblcms2$secondaryArchSuffix = 2.0.8 compat >= 2
+	lcms$secondaryArchSuffix = $portVersion
+	cmd:jpgicc$commandSuffix = 3.2
+	cmd:linkicc$commandSuffix = 2.2
+	cmd:psicc$commandSuffix = 2.1
+	cmd:tificc$commandSuffix = 6.2
+	cmd:transicc$commandSuffix = 4.3
```

Comment:
Shouldn't the cmd's go to a _tools package? Don't think they are really needed for the library?

Reply:
not required, could be another PR as you wish.

Reply:
Or I could move them in this PR